pynestml/transformers/synapse_post_neuron_transformer.py

```diff
@@ -496,7 +529,7 @@ def mark_post_port(_expr=None):
 
         for state_var, alternate_name in zip(post_connected_continuous_input_ports, post_variable_names):
```

Comment:
Since `alternate_name` is not used in the loop anymore, you can just loop over the `post_connected_continuous_input_ports`.